src/test/java/io/openshift/booster/OpenShiftIT.java

```diff
@@ -26,15 +26,15 @@
 public class OpenShiftIT {
     private String project;
 
-    private final String applicationName = "http-vertx";
+    private static final String APPLICATION_NAME = System.getProperty("app.name","http-vertx");
```

Comment:
@akoniar could you please roll back the changes here so we can keep the boosters in sync? Thanks!

Reply:
Strictly speaking they aren't totally in sync anyway, as @akoniar used `application.name` and I used `app.name`.

Reply:
Strictly speaking they aren't totally in sync anyway, as @akoniar used `application.name` and I used `app.name`.

Reply:
Yeah, I just noticed this difference is in a few other boosters as well. I've already merged some PRs though.